communication/zoom-guidelines.md

```diff
@@ -1,65 +1,87 @@
 
-These guidelines are meant as a tool to help Kubernetes members manage their Zoom resources. 
+These guidelines are meant as a tool to help Kubernetes members manage their Zoom resources.
 Check the main [moderation](./moderation.md) page for more information on other tools and general moderation guidelines.
 
+## Current State
+Zoom licenses are managed by the [CNCF Service Desk](https://github.com/cncf/servicedesk) through the Zoom Admins listed below. At the time of this update, we have 41 paid pro user licenses with 38 accounted for.
+
 ## Code of Conduct
 Kubernetes adheres to Cloud Native Compute Foundation's [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) throughout the project, and includes all communication mediums.
 
-## Moderation
+## Obtaining a Zoom License
 
-Zoom has documentation on how to use their moderation tools: 
+Each SIG should have a paid Zoom account that all leads/chairs/necessary trusted owners have access to through their k-sig-foo-leads@googlegroups.com account
+See the [SIG Creation procedure](/sig-governance.md#sig-creation-procedure) document on how to set up an initial account.
 
-- https://support.zoom.us/hc/en-us/articles/201362603-Host-Controls-in-a-Meeting
+## Setting Up Your Meeting and Moderation
+
+Do not share your zoom link on social media.
 
-Check the "Screen Share Controls" (via the ^ next to Share Screen): Select who can share in your meeting and if you want only the host or any participant to be able to start a new share when someone is sharing.  
+Moderation will not be available if you are not following this list:
+* [latest version](https://zoom.us/download)
+* logged in as the leads of that meeting OR have the host key. (Example: you need to use the leads account for sig arch if you are running that meeting or have their meeting key with join before host enabled)
+* using a meeting that was set up through the "Meeting" tab in the zoom account and NOT the personal meeting ID
 
-You can also put an attendee on hold. This allows the host(s) to put attendee on hold to temporarily remove an attendee from the meeting. 
+After the meeting has started:
+* Assign a cohost to help with moderation. It should never be your notetaker unless it's a very small group.
+* Turn off screen sharing for everyone and indicate only host. If you have others that need to share their screen, the host can enable that on the fly. (via the ^ next to Share Screen)
+
+If you're dealing with a troll or bad actor:
+* You can put an attendee on hold. The participant will be put into a 'waiting room' and not have ability to chat or discuss written or verbally until the host undoes the hold.
+* Remove the participant. This is permanent. They will never be able to come back into that meeting ID on that device. Do not joke around with this feature; it's better to use the hold first and then remove.
```

Comment:
Maybe: Please be cautious when testing or using this feature, as it is permanent.